series/supportedseries.go

```diff
@@ -293,6 +294,33 @@ func VersionSeries(version string) (string, error) {
 	return "", errors.Trace(unknownVersionSeriesError(version))
 }
 
+// WindowsVersionSeries returns the series (eg: win2012r2) for the specified version
+// (eg: Windows Server 2012 R2 Standard)
+func WindowsVersionSeries(version string) (string, error) {
```

Comment:
looking more closely at the code it seems to think that it *does* support seriesVersions for the various 'win2008r2' et al, but it does so by mapping them explicitly back onto the exact string:

var seriesVersions = map[string]string{
    "precise":          "12.04",
    "quantal":          "12.10",
    "raring":           "13.04",
    "saucy":            "13.10",
    "trusty":           "14.04",
    "utopic":           "14.10",
    "vivid":            "15.04",
    "wily":             "15.10",
    "xenial":           "16.04",
    "yakkety":          "16.10",
    "zesty":            "17.04",
    "artful":           "17.10",
    "bionic":           "18.04",
    "win2008r2":        "win2008r2",
    "win2012hvr2":      "win2012hvr2",
    "win2012hv":        "win2012hv",
    "win2012r2":        "win2012r2",
    "win2012":          "win2012",
    "win2016":          "win2016",
    "win2016hv":        "win2016hv",
    "win2016nano":      "win2016nano",
    "win7":             "win7",
    "win8":             "win8",
    "win81":            "win81",
    "win10":            "win10",
    "centos7":          "centos7",
    "opensuseleap":     "opensuse42",
    genericLinuxSeries: genericLinuxVersion,
}

that doesn't seem particularly useful.

Why wouldn't we just fix the mapping to just be to the keys in windowsVersions
and then we don't have to do this weird extra functions that give us exact values that we probably would have wanted anyway.

Are the other providers (azure) expecting the short-form for Windows Version?

It just seems a little odd that you couldn't do

v == SeriesVersion(VersionSeries(v))
(I think this actually works because the map is exactly identity
or really
v == SeriesVersion(WindowsVersionSeries(v))

But it also doesn't seem useful to have SeriesVersion(v) == v.

Can we just update them?